From f6f3025232918094b131f96d66ac1abc140e9945 Mon Sep 17 00:00:00 2001
From: Dmitry Lavnikevich <dmitry.lavnikevich@edality.by>
Date: Sat, 7 Apr 2018 15:45:23 +0300
Subject: [PATCH] Remove dependency on generated autoconf.h

This file can be generated in other paths. Yocto generates it under
grapeboard_<config>_config/ path. And this causes build failure.
---
 include/configs/grapeboard.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/include/configs/grapeboard.h b/include/configs/grapeboard.h
index 6782762cc0..437ef89312 100644
--- a/include/configs/grapeboard.h
+++ b/include/configs/grapeboard.h
@@ -14,7 +14,6 @@
 #include <asm/arch/config.h>
 #include <asm/arch/stream_id_lsch2.h>
 #include <asm/arch/soc.h>
-#include <../../../include/generated/autoconf.h>
 
 #define CONFIG_SUPPORT_RAW_INITRD
 
-- 
2.16.2

